siliconcompiler/tools/opensta/timing.py

```
@@ -126,6 +149,43 @@ def add_opensta_poweractivity(self, scope: str, library: str, fileset: str,
+            index: The specific index to apply this configuration to.
+            clobber: If True, overwrites the existing list of skipped reports.
+                     If False, appends to the existing list.
+        """
+        if isinstance(report_type, str):
+            patterns = [report_type]
+        else:
+            patterns = list(report_type)
+
+        if any("*" in pattern for pattern in patterns):
+            enum_type = list(
+                NodeType.parse(BaseSchema.get(self, "var", "skip_reports", field="type")))[0]
```

Ruff (0.15.21)

[warning] Prefer `next(...)` over single element slice

Replace with `next(...)`

(RUF015)